src/frontend/screens/Library/index.tsx

```diff
@@ -515,6 +528,55 @@ export default React.memo(function Library(): JSX.Element {
       )
     }
 
+    return library
+  }, [
+    storesFilters,
+    platformsFilters,
+    epic.library,
+    epic.username,
+    gog.library,
+    gog.username,
+    amazon.library,
+    amazon.user_id,
+    sideloadedLibrary,
+    platform,
+    filterText,
+    showHidden,
+    hiddenGames,
+    showFavouritesLibrary,
+    favouritesIds,
+    currentCustomCategories,
+    customCategories,
+    showInstalledOnly,
+    showNonAvailable,
+    showSupportOfflineOnly,
+    showThirdPartyManagedOnly,
+    showUpdatesOnly,
+    gameUpdates
+  ])
+
+  // select library
+  const libraryToShow = useMemo(() => {
+    let library = [...gamesForAlphabetFilter]
+
+    // Alphabetical filter
+    if (alphabetFilterLetter) {
+      if (alphabetFilterLetter === '#') {
+        const startsWithNumber = /^[0-9]/
+        library = library.filter(
+          (game) => game.title && startsWithNumber.test(game.title)
+        )
+      } else {
+        library = library.filter(
+          (game) =>
+            game.title &&
+            game.title
+              .toLowerCase()
+              .startsWith(alphabetFilterLetter.toLowerCase())
```

Comment:
this still fails to match something like `[REDACTED]` for the R, even though `[REDACTED]` is counted when checking if the `R` is an available character